Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/readme.txt

```diff
@@ -0,0 +1,10 @@
+========================================================================
+    C++/WinRT WinMLPIXSample Project Overview
+========================================================================
+
+This project demonstrates how to get started using PIX for windows to do
+GPU captures as well as setting markers programmatically
+
+========================================================================
+
```

Comment:
Can you add some links to relevant PIX tutorials? Additionally, how do you observe the results of the programmatic capture?

Reply:
Yes, good suggestion